momento-sdk/src/main/java/momento/sdk/CacheClient.java

```diff
@@ -1199,6 +1201,134 @@ public CompletableFuture<CacheDictionarySetFieldResponse> dictionarySetField(
     return scsDataClient.dictionarySetField(cacheName, dictionaryName, field, value, null);
   }
 
+  /**
+   * Sets all the fields in the given dictionary.
+   *
+   * @param cacheName - The cache containing the list.
+   * @param dictionaryName - The dictionary to set the field in.
+   * @param items - The fields to set.
+   * @param ttl Time to Live for the item in Cache. This ttl takes precedence over the TTL used when
+   *     building a cache client {@link CacheClient#builder(CredentialProvider, Configuration,
+   *     Duration)}
+   * @return Future containing the result of the dictionary fetch back operation.
+   */
+  public CompletableFuture<CacheDictionarySetFieldsResponse> dictionarySetFieldsStringString(
+      String cacheName, String dictionaryName, Map<String, String> items, CollectionTtl ttl) {
```

Comment:
I think we are standardizing on the word element for items in a collection and value for an individual item. We should probably change this to elements to match .NET.

Reply:
We can also make the language more consistent on a future pass, since I assume it is already inconsistent everywhere in CacheClient.

Reply:
Will do in the next PR.